WTWSMS/decisions/WtWSMS_realm_decisions.txt

```diff
@@ -73,7 +73,15 @@ decisions = {
 	support_greens = {
 		only_playable = yes
 		potential = {
-			culture_group = latin
+			OR = {
+				culture_group = latin
+				culture = roman
+				has_landed_title = e_wre
+				has_landed_title = e_roman_empire
+				has_title = c_roma
+				has_title = b_rome
+				has_title b_roma
+			}
```

Comment:
Why? If you control Rome, why wouldn't you participate in YOUR CITY'S greatest spectacle? You don't think a foreign Roman governor wouldn't want to see a chariot race and take sides? Anyone who controls Rome when Rome has chariot races is going to engage in them to get out among the people and try to convince him he is tolerant of their ways.